deploy/crds/toolchain_v1alpha1_useraccount.yaml

```diff
@@ -11,6 +11,27 @@ spec:
     kind: UserAccount
     plural: useraccounts
   scope: Namespaced
+  additionalPrinterColumns:
```

Comment:
This file is generated after https://github.com/codeready-toolchain/api/blob/master/pkg/apis/toolchain/v1alpha1/useraccount_types.go, and I'm afraid these new additions to this yaml file might be lost next time we make an update in the API.

Reply:
Hm.. right @xcoulon. I missed that important aspect. We will have to do something in our generator (or at least document it for now somewhere) to avoid that.

But these additions are important. We have to keep them in CRD.